mimic/rest/maas_api.py

```diff
@@ -810,13 +867,12 @@ def get_alarm(self, request, tenant_id, entity_id, alarm_id):
         entities = self._entity_cache_for_tenant(tenant_id).entities
 
         try:
-            _assert_alarm_exists(entities, entity_id, alarm_id)
+            alarm = _get_alarm(entities, entity_id, alarm_id)
+            return json.dumps(alarm.to_json())
         except (ObjectDoesNotExist, ParentDoesNotExist) as e:
```

Comment:
This pattern seems to be repeated a few times, and it would be good to refactor it at some point as well.